web/src/main/webapp/heavyhittersview.ts

```diff
@@ -0,0 +1,62 @@
+import {significantDigits, FullPage} from "./ui";
+import {ColumnDescription, RemoteTableObjectView, Schema, ContentsKind} from "./tableData";
+import {TableDataView} from "./table"
+import {Converters} from "./util";
+
+
+export class HeavyHittersView extends RemoteTableObjectView {
+    constructor(remoteObjectId: string, page: FullPage) {
+	super(remoteObjectId, page);
+	this.topLevel = document.createElement("div");
+	var heading = document.createElement("h1");
+	var h_text = document.createTextNode("Top rows in Sorted Order");
+	heading.appendChild(h_text);
+	this.topLevel.appendChild(heading);
+    }
+
+    refresh(): void {}
+    
+    public fill(tdv: TableDataView, schema: Schema): void{
+	let table = document.createElement("table");
+	this.topLevel.appendChild(table);
+	let tHead = table.createTHead();
+	let thr = tHead.appendChild(document.createElement("tr"));
+	for (let i =0; i < schema.length; i++) {
+	    let cd = new ColumnDescription(schema[i]);
+            let thd = document.createElement("th");
+            thd.innerHTML = cd.name;
+            thr.appendChild(thd); 
+	}
+	let thd1 = document.createElement("th");
+        thd1.innerHTML = "Counts";
+        thr.appendChild(thd1);
+	let thd2 = document.createElement("th");
+        thd2.innerHTML = "Percentages";
+        thr.appendChild(thd2);
+	
+	let tBody = table.createTBody();
+        if (tdv.rows != null) {
+            for (let i = 0; i < tdv.rows.length; i++) {
+		let trow = tBody.insertRow();
+		for (let j = 0; j < schema.length; j++) {
+		    let cd = schema[j];
+		    let cell = trow.insertCell(j);
+		    //cell.classList.add(this.columnClass(cd.name));
+		    cell.style.textAlign = "right";
+		    let value = tdv.rows[i].values[j];
+		    if (value == null) {
+			cell.classList.add("missingData");
+			cell.textContent = "missing";
+		    } else 
+			cell.textContent = String(value); //TODO
+		}
+		let cell1 = trow.insertCell(schema.length);
+		cell1.style.textAlign = "right";
+		cell1.textContent = String(tdv.rows[i].count);
```

Comment:
Consider using the same kind of display for count as in the TableDataView, with a horizontal bar.